tuned/plugins/plugin_disk.py

```diff
@@ -142,6 +158,9 @@ def _drive_spinning(self, device):
 		return not "standby" in out and not "sleeping" in out
 
 	def _instance_update_dynamic(self, instance, device):
+		if not device in self._hdparm_devices:
```

Comment:
Can we log it just once in the `_instance_apply_dynamic` and maybe log it at the level info.